include/prism.h

```diff
@@ -259,11 +259,11 @@ PRISM_EXPORTED_FUNCTION void pm_dump_json(pm_buffer_t *buffer, const pm_parser_t
  * In order to parse Ruby code, the structures and functions that you're going
  * to want to use and be aware of are:
  *
- * * `pm_parser_t` - the main parser structure
- * * `pm_parser_init` - initialize a parser
- * * `pm_parse` - parse and return the root node
- * * `pm_node_destroy` - deallocate the root node returned by `pm_parse`
- * * `pm_parser_free` - free the internal memory of the parser
+ * * <code>#pm_parser_t</code> - the main parser structure
+ * * `pm_parser_init()` - initialize a parser
+ * * `pm_parse()` - parse and return the root node
+ * * `pm_node_destroy()` - deallocate the root node returned by `pm_parse()`
```

Comment:
Hmmm, I can't find `pm_node_destroy` in the docs (and neither can Doxygen. Any clue what's up with that?

Reply:
Don't know